website/content/docs/internals/telemetry.mdx

```diff
@@ -150,10 +150,12 @@ These metrics cover measurement of token, identity, and lease operations, and co
 | `vault.expire.lease_expiration`                                                                 | Count of lease expirations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | leases   | counter |
 | `vault.expire.job_manager.total_jobs`                                                           | Total pending revocation jobs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | leases   | sample  |
 | `vault.expire.job_manager.queue_length`                                                         | Total pending revocation jobs by auth method                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | leases   | sample  |
+| `vault.expire.job_manager.workers` (cluster, queue_id)                                          | Count of expiration workers per auth mount                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | leases   | counter |
```

Comment:
couple things to point out for reviewers
1. Is `queue_id` the correct term to use here? ( as opposed to `mount_point`, `auth_method` or `mount_id` )
2. Is `leases` the correct unit ? Technically it should be workers, but that won't mean much externally, so not sure.

Reply:
removed in 1df235f